### PR TITLE
Feat: add capitalize keyword to avoid lint error

### PR DESCRIPTION
**Pull request by noahlias**

### CODE REVIEW

**Comment by noahlias** [comment]
add feature about the extra keywords, the last commit of Blak project has commit msg error

File: hooks/check_commit_msg.py [hunk 1]
```diff
@@ -23,10 +23,11 @@ def main(*args):
         data = FILE.read()
         msg = data.split('\n')[0]
         keyword = msg.split(':', maxsplit=1)[0]
-        if keyword not in keywords:
+        all_keyword = keywords + [s.capitalize() for s in keywords]
+        if keyword not in all_keyword:
             logger.info(
                 "\tCommit message must start with a keyword"
-                f"\n\tvalid keywords {*keywords,}\n"
+                f"\n\tvalid keywords {*all_keyword,}\n"
```
Comment (p0lygun) [inline hooks/check_commit_msg.py]:
```suggestion
                f"\n\tvalid keywords {*keywords,}\n"
```
Don't show all keywords here just the normal ones or the capitalized ones

Comment (p0lygun) [reply hooks/check_commit_msg.py]: as showing all will show a very long list

File: hooks/check_commit_msg.py [hunk 2]
```diff
@@ -23,10 +23,11 @@ def main(*args):
         data = FILE.read()
         msg = data.split('\n')[0]
         keyword = msg.split(':', maxsplit=1)[0]
-        if keyword not in keywords:
+        all_keyword = keywords + [s.capitalize() for s in keywords]
```
Comment (p0lygun) [inline hooks/check_commit_msg.py]: should we all add all caps ? 
so then all these will be Valid 
`feat`, `Feat`, `FEAT`

**Review by p0lygun — CHANGES_REQUESTED** [review]

File: hooks/check_commit_msg.py (hunk 2 above)
Comment (noahlias) [reply hooks/check_commit_msg.py]: image
not all , only Feat and feat

Comment (p0lygun) [reply hooks/check_commit_msg.py]: No I'm asking if we should also add FEAT

Comment (noahlias) [reply hooks/check_commit_msg.py]: or do we specify the first character in uppercase?

Comment (p0lygun) [reply hooks/check_commit_msg.py]: Ok then add FEAT to valid keyword,
So we have feat, Feat, FEAT all these three are valid

**Comment by p0lygun** [comment]
LGTM!

**Pull request merged**